ethcore/sync/src/chain/mod.rs

```diff
@@ -1328,8 +1328,8 @@ impl ChainSync {
 	}
 
 	/// Broadcast private transaction message to peers.
-	pub fn propagate_private_transaction(&mut self, io: &mut SyncIo, transaction_hash: H256, packet_id: PacketId, packet: Bytes) {
-		SyncPropagator::propagate_private_transaction(self, io, transaction_hash, packet_id, packet);
+	pub fn propagate_private_transaction(&mut self, io: &mut SyncIo, transaction_hash: H256, protocol: ProtocolId, packet_id: PacketId, packet: Bytes) {
```

Comment:
I think they only use `WARP_SYNC_PROTOCOL`, right?

Reply:
In fact, the calling functions specify `WARP_...`, yes, and it looks like they're only called in one place. Maybe we can move the constant here.